patientslikeme/drugsDict.py

```diff
@@ -0,0 +1,76 @@
+from bs4 import BeautifulSoup
+import requests
+import urllib
+
+def initializeSoup(url):
+    request = requests.get(url)
+    soup = BeautifulSoup(request.text, 'html.parser')
+    return soup
+
+def getDrugName(url):
+    soup = initializeSoup(url)
+    #removes js and css
+    for script in soup (["script", "style"]):
+        script.extract()
+    return soup.h1.span.string
+    
+def addBrandNames(url,dictionary):
+    soup = initializeSoup(url)
+    brandNames = soup.find("span", {"class":"comma-separated"})
+    if(brandNames is not None):
+        brandNames = brandNames.find_all('a')
+        #print(brandNames)
+        for brand in brandNames:
+            url = brand.get("href")
+            fullUrl = "https://www.patientslikeme.com" + url
+            name = brand.string.rstrip().encode('utf-8')
+            dictionary[name] = fullUrl
+
+#this will be our drug dicitonary
+drugDict = {}
+website = "https://www.patientslikeme.com/treatments/browse?cat="
+
+#creating dictionary for drugs
+drugDict = {}
+allCat = [1,2]
```

Comment:
okay I gave fixed the issue by creating constants of each category and assigning them with the appropriate numbers

https://gyazo.com/00587a63888c70aeb0339c04b77a89c3